source/less/blocks/product-item.less

```
@@ -1,12 +1,121 @@
+.product-item {
+  border-top: 1px solid @special-grey;
+  box-sizing: border-box;
+
+  &:last-child {
+    border-bottom: 1px solid @special-grey;
+  }
+
+  @media (min-width: @tablet-width) {
+    margin: 0;
+  }
+
+  @media (min-width: @desktop-width) {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+}
+
+.product-item__info {
+  @media (min-width: @tablet-width) {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    box-sizing: content-box;
+  }
+
+  @media (min-width: @desktop-width) {
+    flex-grow: 1;
+  }
+}
+
 .product-item__name {
+  margin: 13px 0 0 0;
+  padding-left: 1px;
   font-family: @font-oswald;
   font-weight: 400;
   font-size: 16px;
   line-height: 16px;
   text-transform: uppercase;
   color: @special-dark-grey-2;
+
+  @media (min-width: @tablet-width) {
+    margin: 25px 0;
+    padding: 0 0 0 38px;
+    flex-grow: 1;
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  @media (min-width: @desktop-width) {
+    min-width: 245px;
+    margin: 0 80px 0 0;
+  }
+}
+
+.product-item__table {
+  min-width: 100%;
+  margin: 13px -2px 15px -2px;
+
+  @media (min-width: @tablet-width) {
+    min-width: 50%;
+    margin: 0;
```

```suggestion
    margin: 0 0 0 auto;
```

надо оттолкнуть в правый край, чтобы цена прилипала к правой границе